Makefile

```diff
@@ -10,5 +10,12 @@ all:
 	# Get rid of the unnecessary files
 	find lubuntu/ -type f -iname "*.in" | xargs rm -f
 	find lubuntu/ -type f -iname "Makefile" | xargs rm -f
+	(cd cinnamon-remix/po/ && make)
```

Comment:
Now that we have one more flavor, I think we should simplify this (Bash for loop?).